recipe/psycopg2-binary/setup.py

```diff
@@ -0,0 +1,12 @@
+import os
+from setuptools import setup
+
+
+version = os.environ['PKG_VERSION']
```

Comment:
Hm... maybe treat this as a template and fill it in explicitly? i mean, nobody really expects to _run_ `setup.py` after the fact, but it would _for sure_ fail...

Reply:
or ship the old `__import__('setuptools').setup()` and generate a `setup.cfg`...

Reply:
Feel free to implement this on a branch on your fork and I'll be happy to cherry-pick your suggested changes over here.  I've put in the time I have for this project.  I need to get back to my own work...

Reply:
@bollwyvl, I downloaded the current conda-forge package for `psycopg2` and I don't see anywhere that `setup.py` would be retained in the package.  So it's not just a question of a user not being likely to call `setup.py`, it's not available even if they wanted to (in contrast to a pypi package, for example).  The conda recipe is retained but not in a form where a user would try to run it, just for provenance, I presume.

Are you sure this is really an issue that needs solving?

Reply:
that's good data!